tools/accuracy_checker/openvino/tools/accuracy_checker/metrics/image_quality_assessment.py

```diff
@@ -478,3 +478,30 @@ def _create_loss(self):
                 sl._modules[module_id] = feats[int(module_id)] # pylint: disable=W0212
             setattr(loss.net, 'slice{}'.format(slice_id), sl)
         return loss
+
+
+class PeakSignalToNoiseRatioIFRNet(BaseRegressionMetric):
```

Comment:
The same comment for naming. What the difference with original psnr metric?
Can it be just configured for getting desired behaviour by parameters instead of introducing new metric?

Reply:
Reused PSRN with added 'unweighted_average' parameter to calculate psnr as for grayscale image.